reverse_strand.py

```diff
@@ -0,0 +1,35 @@
+#! /usr/bin/python3
+import argparse
+from argparse import RawTextHelpFormatter
+import bamnostic as bs
+
+description="Reverses the allele for variants that got mapped onto the reverse strand of the new genome, and prints everything to a new file\nAlso filters out the non-mapped"
+
+parser = argparse.ArgumentParser(description = description, formatter_class=RawTextHelpFormatter)
+parser.add_argument("-i", "--bam", help="bam file containing remapped variants ")
+parser.add_argument("-o", "--outfile", help="name of new file")
+args = parser.parse_args()
+
+bamfile=bs.AlignmentFile(args.bam, 'rb')
+outfile=open(args.outfile, 'w')
+
+#reverses the allele for variants that got mapped onto the reverse strand of the new genome, and prints everything into correct columns
+
+for read in bamfile:
+	name=read.read_name
+	info=name.split("|")
+	#mapped onto reverse strand:
+	if read.flag & 16: #decoding bitwise flag with &; may need to be changed back to ==
+		if info[2]=='A':
```

Comment:
My suggestion is to assign the reverse nucleotide in the if-else, and take the long `write` call out of them to avoid repetition.

Reply:
Good suggestion, added.

Reply:
@mistyskye Although this discussion is marked as resolved, I can still see two `outfile.write()` calls in this PR. Can you please check if you have pushed all of the latest commits?

Reply:
Oh I didn't change the else part, this has been fixed.